lib/stripe/errors.rb

```diff
@@ -14,16 +14,18 @@ class StripeError < StandardError
     attr_reader :http_headers
     attr_reader :http_status
     attr_reader :json_body # equivalent to #data
+    attr_reader :code
```

Comment:
Nit, but mind just keeping this list alphabetized? Also (as an aside), what do you think about the comment above this about preferring `response`? Should we just take that out at this point?

Reply:
Ah, good point, I completely missed that comment 🙈 

Since `code` is specific to errors and not to all Stripe responses, should I move it away from those parameters, with its own comment?

Reply:
> Since code is specific to errors and not to all Stripe responses, should I move it away from those parameters, with its own comment?

Honestly ... I might just leave it, it's too weird to have accessors for basically everything, but minus a couple fields. I was more wondering whether we should just remove this comment completely and accept that these fields are probably going to be around for a while given that there's not much cost to maintaining them anyway.

Reply:
Fair enough :)